MongoDB/recipes/default.rb

```diff
@@ -0,0 +1,37 @@
+#
+# Cookbook:: MongoDB
+# Recipe:: default
+#
+# Copyright:: 2017, The Authors, All Rights Reserved.
+
+#'MongoDB::importpkey'
+apt_repository 'import_key' do
+  uri 'hkp://keyserver.ubuntu.com:80'
+  components ['main']
+  distribution 'trusty'
+  key '7F0CEB10'
+  keyserver 'keyserver.ubuntu.com'
+  action :add
+  deb_src true
+end
+
+#'MongoDB::createfile'
+apt_repository 'create_file' do
```

Comment:
why are we using apt_repository multiple twice? Only one should do. Please check this on new AWS instance